src/psyclone/tests/psyad/main_test.py

```diff
@@ -624,7 +624,17 @@ def test_main_otest_lfric(tmpdir, capsys):
           "-otest", harness_out])
     output, error = capsys.readouterr()
     assert error == ""
-    assert output == ""
+    # TODO 2668: This deprecation of options causes this test to see the
+    # deprecation message.
+    correct = '''Deprecation Warning: The options parameter to \
+Transformation.validate is now deprecated. Please use the individual \
+arguments, or unpack the options with **options. \
+See the documentation for more details.
+Deprecation Warning: The options parameter to Transformation.validate \
+is now deprecated. Please use the individual arguments, or unpack the \
+options with **options. See the documentation for more details.
```

Comment:
This makes me think we should write a short paragraph (probably in the user guide section on Transformations) about the move from the `options` dict to having named arguments. We could give an example of the old and the new syntax.

Reply:
Added documentation here, which should guide people towards the new solution as Transformations support it.